supadata/youtube.py

```
@@ -183,6 +188,27 @@ def playlist(self):
             self._playlist_instance = self._Playlist(self)
         return self._playlist_instance
 
+    @property
+    def video(self):
```

Ruff (0.8.2)

Redefinition of unused `video` from line 134

(F811)